include/swift/AST/DiagnosticsCommon.def

```diff
@@ -145,7 +145,7 @@ NOTE(fixit_correct_source_location_file,none,
 
 // Usually, but not always, emitted from the driver
 ERROR(error_two_files_same_name,none,
-      "filename \"%0\" used twice: '%1' and '%2'",
```

Comment:
To my understanding, the apostrophe is used for indicating code voice in diagnostics; I think it's reasonable for file names to retain their double quotation marks.

Reply:
Sorry for this late response!

In my opinion, the distinction between code voice and file names are not that clear, especially when it comes to file paths. However, I can see the point you might be making (paraphrasing it to make sure I understand correctly): a file name does not originate from a code context, so even if it used in code (or not, as in the case of this marked line) it should not be treated as code voice. Is this what you mean?

Reply:
Here's how I would format the same sentence when writing on GitHub:

> filename "foobarbaz.png" used twice

I wouldn't use code voice. That is, I wouldn't write...

> filename `foobarbaz.png` used twice

...nor would I write...

> filename `"foobarbaz.png"` used twice

For the purposes of Swift diagnostics, we use single quotes instead of the backtick for code voice, but I wouldn't use code voice here.

Reply:
I understand this part. 

But I question if it is indeed the case that file names should use double quotes. There are many places where file names are already using single quotes, including the diagnostic message on [line 35 of the same file](https://github.com/apple/swift/blob/e3fece033c8e764004abf40c381846af16bfcf88/include/swift/AST/DiagnosticsCommon.def#L39).

Reply:
🤷 In either case, it's fair to say that file names aren't "symbols" and if no consensus we can subset those changes out of this PR.

Reply:
Sounds reasonable to me. I will remove my changes to file names' quotations.

Reply:
done in f6fd93781670a1fe5ec2ebe3e8fb5107f43922cc